packages/core/admin/server/controllers/webhooks.js

```diff
@@ -1,31 +1,28 @@
 'use strict';
 
 const _ = require('lodash');
-const { yup, webhook: webhookUtils, validateYupSchema } = require('@strapi/utils');
+const { yup, validateYupSchema } = require('@strapi/utils');
 
 const urlRegex =
   /^(?:([a-z0-9+.-]+):\/\/)(?:\S+(?::\S*)?@)?(?:(?:[1-9]\d?|1\d\d|2[01]\d|22[0-3])(?:\.(?:1?\d{1,2}|2[0-4]\d|25[0-5])){2}(?:\.(?:[1-9]\d?|1\d\d|2[0-4]\d|25[0-4]))|(?:(?:[a-z\u00a1-\uffff0-9_]-*)*[a-z\u00a1-\uffff0-9_]+)(?:\.(?:[a-z\u00a1-\uffff0-9_]-*)*[a-z\u00a1-\uffff0-9_]+)*\.?)(?::\d{2,5})?(?:[/?#]\S*)?$/;
 
-const webhookValidator = yup
-  .object({
-    name: yup.string().required(),
-    url: yup.string().matches(urlRegex, 'url must be a valid URL').required(),
-    headers: yup.lazy((data) => {
-      if (typeof data !== 'object') {
-        return yup.object().required();
-      }
-
-      return yup
-        .object(
-          _.mapValues(data, () => {
-            yup.string().min(1).required();
-          })
-        )
-        .required();
-    }),
-    events: yup.array().of(yup.string().oneOf(_.values(webhookUtils.webhookEvents)).required()),
-  })
-  .noUnknown();
+const webhookValidator = yup.object({
+  name: yup.string().required(),
+  url: yup.string().matches(urlRegex, 'url must be a valid URL').required(),
+  headers: yup.lazy((data) => {
+    if (typeof data !== 'object') {
+      return yup.object().required();
+    }
+
+    return yup
+      .object(
+        _.mapValues(data, () => {
+          yup.string().min(1).required();
+        })
+      )
+      .required();
+  }),
+});
```

Comment:
Why did you remove the attributes of the events from being validated ?

I think we should still validate the shape of the input data (having an array of strings in events). And the content of the events array should be validated in the service as it is a business rule.